docs/content/users/quickstart.md

```diff
@@ -342,6 +342,76 @@ Here's a quickstart instructions for a number of different environments:
     This way we only need to change the value of `DB_CONNECTION` to `ddev` in the `.env` to work with the `db` service.
     This is very handy if you have a local database installed and you want to switch between the connections faster by changing only one variable in `.env`
 
+=== "Craft CMS"
+
+    ## Craft CMS
+
+    [Craft CMS](https://craftcms.com) can be downloaded with [Composer](https://craftcms.com/docs/4.x/installation.html#downloading-with-composer), by cloning a [Craft starter project](https://github.com/craftcms/craft) or by [manually downloading](https://craftcms.com/docs/4.x/installation.html#downloading-an-archive-file-manually) a zipped archive. 
+    
+    === "New projects"
+    
+        ### Composer build
+        
+        Use this to create a new Craft CMS project from the official [Craft starter project](https://github.com/craftcms/craft) or a third-party starter project using Composer.
+        
+        ```bash
+        mkdir path/to/my-project
+        cd path/to/my-project
+        composer create craftcms/craft . --no-install --no-scripts
+        ddev config --project-type=craftcms
+        ddev start
```

Comment:
```suggestion
        ddev config --project-type=craftcms
        ddev composer create craftcms/craft  --no-install --no-scripts
        ddev composer install
```

Reply:
So we looked at doing it this way, but decided against it. Because if we do it this way, the `.env` and `.env.example` and `.env.example.dev` will not be present, so `ddev config --project-type=craftcms` will do nothing in terms of being able to automatically update the `.env` file for you.

Recommend not changing this.

Reply:
This will have to be sorted out. Composer and PHP are not required to use ddev.